### DISCUSSION TITLE
Changing gravity in different locations with similar axis coordinates

### DISCUSSION THREAD

**Discussion by warcraft2a** [firemodels/fds#11949] [Q&A]
Dear FDS community and developers,

I am trying to model a tunnel which after 1000 meters splits into two tunnels. The common section and the two splitting tunnels each have their own slopes. Using the gravity ramp, it is possible to model the slope of the common section with no problem. However, when it comes to modelling the slope of the two splitting tunnels, I have a problem that each tunnel has its own slope, which means its own gravity ramp that needs to be used for the same axis coordinates.

Is there a way to assign different gravity ramps to different meshes? Can the fact that gravity is applied universally in FDS be changed?
Aside from modelling the downward and upward slope of the tunnel by modifying geometry, is there another approach to model the effects of the slope of this tunnel?

Thank you for your help and time to consider this topic.

Kind regards,
Alireza

**Comment by drjfloyd** [comment]
This cannot be done. Just make your geometry such that is follows the tunnel slope and use mulitple meshes.  The gravity feature dates back to very early versions of FDS when you could only have one MESH. There it was punishing to deal with an significant change in tunnel elevation as your single MESH meant many wasted grid cells.
